baybe/simulation.py

```diff
@@ -3,9 +3,9 @@
 The term "simulation" can have two slightly different interpretations, depending on the
 applied context:
 
-*   It can refer to "backtesting" a particular DOE strategy on a fixed (finite)
+*   It can refer to "backtesting" a particular DOE recommender on a fixed (finite)
```

Comment:
This also does not refer to the object but the abstract concept I think.

Reply:
yes, changed back